azure/src/main/java/sunstone/azure/impl/AzureSunstoneDeployer.java

```diff
@@ -45,15 +46,15 @@ private void deployArmTemplate(WithAzureArmTemplate armTemplateDefinition, Azure
         String content = null;
         try {
             content = getResourceContent(armTemplateDefinition.template());
-            String group = resolveOrGetFromSunstoneProperties(armTemplateDefinition.group(), AzureConfig.GROUP);
+            String group = armTemplateDefinition.group().isEmpty() ? SunstoneConfig.getString(AzureConfig.GROUP) : SunstoneConfig.resolveExpressionToString(armTemplateDefinition.group());
```

Comment:
Expression on left seems simpler. 

Can we introduce similar metod in `SunstoneConfig`?

Reply:
IMHO SunstineConfig is about retrieving properties and having it encapsulate such functionality may be confusing.

And I moved from `resolveOrGetFromSunstoneProperties` because I forgot what the function did and I had to look into it. Now, it is longer - yes - but IMO more readable.

Reply:
Ok. This can be marked as Resolved